bower.json

```diff
@@ -1,8 +1,9 @@
 {
   "name": "monocle-client-js",
-  "version": "0.0.3",
+  "version": "0.0.4",
```

Comment:
Don't bump version numbers until after merging (there may be other PRs that get merged before a release is tagged/published)